src/app/lib/config/services/hausa.js

```diff
@@ -160,16 +154,16 @@ const service = {
         href: 'https://www.bbc.com/usingthebbc/cookies/',
         text: 'Cookies',
       },
-      {
-        href: 'https://www.bbc.com/accessibility/',
-        text: 'Accessibility Help',
-      },
       {
         href: 'https://www.bbc.com/contact/',
         text: 'Contact the BBC',
       },
     ],
     copyrightText: 'BBC. BBC ba tada alhaki game da shafukan da ba nata ba',
+    externalLink: {
+      href: 'https://www.bbc.co.uk/help/web/links/',
+      text: 'Read about our approach to external linking.',
```

Comment:
Hiya, in the translation doc this string is "Karanta hanyoyin da muke bi dangane da adireshin waje."